pages/mes-jeunes/[jeune_id]/favoris.tsx

```diff
@@ -90,21 +96,40 @@ export const getServerSideProps: GetServerSideProps<FavorisProps> = async (
     return { redirect: sessionOrRedirect.redirect }
   }
   const {
-    session: { accessToken },
+    session: {
+      accessToken,
+      user: { id },
+    },
   } = sessionOrRedirect
 
+  const jeunesService = withDependance<JeunesService>('jeunesService')
   const favorisService = withDependance<FavorisService>('favorisService')
 
   const jeuneId = context.query.jeune_id as string
 
+  const [beneficiaire] = await Promise.all([
+    jeunesService.getJeuneDetails(jeuneId, accessToken),
+  ])
+
+  if (!beneficiaire) {
+    return { notFound: true }
+  }
+
+  const lectureSeule = beneficiaire.idConseiller !== id
```

Comment:
pas besoin de destructurer ligne 101 pour que ce soit bien explicite (à quel id on se réfère)
```suggestion
  const lectureSeule = beneficiaire.idConseiller !== user.id
```
à répercuter sur les autres fichiers